account_invoice_open_sale/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Open sale order from invoice
+============================
+
+This module shows a button to navigate to a customer invoice's related sale
+order if there is one.
+
+Configuration
+=============
+
+The button is only visible for users that have permissions in the sales domain.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/147/8.0
```

Comment:
Hoping that you will want to post this to the OCA repo, can you split up the README accoding to the new conventions?

See https://github.com/OCA/maintainer-tools/tree/master/template/module/readme
(on the whole, this module template repo is useful https://github.com/OCA/maintainer-tools/tree/master/template/module).